tests/test_matrix.py

```diff
@@ -14,6 +17,38 @@ def test_strMatrix():
     assert mat.__str__() == "[{1.0}+{x},{2.0};{3.0},{4.0}]"
 
 
+def test_traceMat():
+
+    mat = getTokens("[1, 2; \
+                      3, 4]")
+    Matrix.isSquare(mat)
+    trace = SquareMat.traceMat(mat)
```

Comment:
Since `traceMat()` is a class method you can use `mat.traceMat()`. Same in line 31.
```suggestion
    trace = mat.traceMat()
```
Also make similar changes in lines 38, 42 (`mat.isSquare()`) and 48 (`mat.transposeMat()`)

Reply:
@8hantanu I tried using the same, but got an error as seen in the screenshot. Do I need to import anything to make this work? And, is the method that I used wrong? 
![hi](https://user-images.githubusercontent.com/39584067/54088358-4f328b00-4383-11e9-813a-ce89e3cf3d75.PNG)

Reply:
@8hantanu Please guide me how to go on about this issue.

Reply:
I have tested the above code with changes I suggested. It is working fine without any errors.